### ISSUE TITLE
Ventilation and infiltration definitions in building.py

### ISSUE THREAD

**Issue by jmrk87** [ladybug-tools/uwg#253]
Hi,
looking at building.py, I found these args definitions in Building():
        infil: Infiltration rate (ACH).
        vent: Ventilation rate (ACH).
However, in Building() properties:
        * vent -- Ventilation (m^3/s/m^2)
The same unit of measure is used for vent in SchDef. Then, it could be a mistake regarding the unit of measure of "vent".

Looking at the code in building.py, it would seems that this second definition is the correct one, indeed:
        # total vent volumetric flow [m3 s-1]
        volVent = self.vent * self.nFloor
        volInfil = self.infil * UCM.bldHeight / \
            3600.  # Change of units AC/H -> [m3 s-1]

However, why using a different formulation for "infill" and "vent"?

Many thanks,
G.

**Comment by saeranv** [comment]
@jmrk87 

Yes that's a typo, the `vent` metric should be [m3 s-1 m-2]. Also the subsequent `volVent` and `volInfil` metric should actually be [m3 s-1 m-2] as well, since the building energy balance (and I think canyon as well) is normalized by floor area, or else those volumetric derivations don't make sense. 

The different metrics are just due to different conventions used by the original DOE reference buildings.
